### PR TITLE
feat: remove liquity event trove usages

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #(issue_number)

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

File: frontend/app/src/components/defi/loan/loans/LiquityLending.vue [hunk 1]
```diff
@@ -73,11 +70,11 @@ const { tc } = useI18n();
...
-          <liquity-trove-events
-            v-else
-            :events="loan.events"
-            :loading="loadingEvents"
-          />
+          <!--          <liquity-trove-events-->
+          <!--            v-else-->
+          <!--            :events="loan.events"-->
+          <!--            :loading="loadingEvents"-->
+          <!--          />-->
```
Comment (kelsos) [inline frontend/app/src/components/defi/loan/loans/LiquityLending.vue:77]: Why are we keeping this comment?

**Comment by kelsos** [comment]
@lukicenturi can you give some more information about this PR?

Are the events removed from the API? 
Do we have any replacements already?

**Comment by yabirgb** [comment]
This PR shouldn't be removing them. It should be querying as we do in the staking section the decoded ethereum events that are deposits/withdrawals

**Comment by kelsos** [comment]
@yabirgb as we discussed in the standup everything that @lukicenturi to complete this issue should be already in develop, so we should be able to proceed with this right?

**Comment by yabirgb** [comment]
yes, we should filter the events by protocol liquity and include the ones with events withdrawal, spend and deposit

**Comment by lukicenturi** [comment]
Okay thanks for the information, I will update this PR soon

**Review by kelsos — APPROVED** [review]
lgtm thanks @lukicenturi

**Pull request merged**